org.gnome.Evince.json

```diff
@@ -12,14 +12,16 @@
         "--filesystem=host",
         "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
         "--metadata=X-DConf=migrate-path=/org/gnome/evince/",
-        "--env=GDK_PIXBUF_MODULE_FILE=/app/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
         "--talk-name=org.gtk.vfs",
         "--talk-name=org.gtk.vfs.*",
         "--talk-name=org.gnome.SessionManager",
+        "--talk-name=org.freedesktop.FileManager1",
         "--own-name=org.gnome.evince",
-        "--own-name=org.gnome.evince.Daemon",
-        /* Commit 8f428fd7683765dd706da06e9f376d3732ce5c0c required for dbus service exports matching own name above */
-        "--require-version=0.11.6"
```

Comment:
Thank you for the PR.

This change has nothing to do with adding x-checker-data. Please remove it from this PR.